### ISSUE TITLE
CLI: --help crashes the application

### ISSUE THREAD

**Issue by rrooij** [asticode/go-astisub#28]
```
~/git/go-astisub ./main  --help 
Usage of ./main:
  -f duration
    	the fragment duration
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x8 pc=0x6c0974]

goroutine 1 [running]:
github.com/asticode/go-astikit.FlagStrings.String(...)
	/home/user/go/pkg/mod/github.com/asticode/go-astikit@v0.2.0/flag.go:34
flag.isZeroValue(0xc0000d6400, 0x0, 0x0, 0x80b9b2)
	/usr/lib/go-1.14/src/flag/flag.go:458 +0x104
flag.(*FlagSet).PrintDefaults.func1(0xc0000d6400)
	/usr/lib/go-1.14/src/flag/flag.go:521 +0x20b
flag.(*FlagSet).VisitAll(0xc0000b6120, 0xc000125b40)
	/usr/lib/go-1.14/src/flag/flag.go:388 +0x61
flag.(*FlagSet).PrintDefaults(0xc0000b6120)
	/usr/lib/go-1.14/src/flag/flag.go:504 +0x4e
flag.PrintDefaults(...)
	/usr/lib/go-1.14/src/flag/flag.go:555
flag.glob..func1()
	/usr/lib/go-1.14/src/flag/flag.go:583 +0xe6
flag.commandLineUsage()
	/usr/lib/go-1.14/src/flag/flag.go:1021 +0x27
flag.(*FlagSet).usage(0xc0000b6120)
	/usr/lib/go-1.14/src/flag/flag.go:884 +0x2f
flag.(*FlagSet).parseOne(0xc0000b6120, 0xc000125dc8, 0xdf3f8fad, 0xce75dcd2148b0b58)
	/usr/lib/go-1.14/src/flag/flag.go:926 +0x1d4
flag.(*FlagSet).Parse(0xc0000b6120, 0xc0000a6030, 0x1, 0x1, 0x1, 0x80b9b2)
	/usr/lib/go-1.14/src/flag/flag.go:971 +0x62
flag.Parse(...)
	/usr/lib/go-1.14/src/flag/flag.go:999
main.main()
	/home/user/git/go-astisub/astisub/main.go:24 +0x122
```

**Comment by asticode** [comment]
Thanks for reporting that issue, I've pushed a fix in `master` and created a `v0.6.0` tag that fixes the issue.

Can you confirm ?

Cheers

**Comment by rrooij** [comment]
Fixes the problem indeed!
